crates/handler/src/execution.rs

```diff
@@ -31,7 +31,9 @@ pub fn create_init_frame(tx: &impl Transaction, spec: SpecId, gas_limit: u64) ->
                     tx.caller(),
                     tx.value(),
                     gas_limit,
-                    EOFCreateKind::Tx { initdata: input },
+                    EOFCreateKind::Tx {
+                        initdata: input.clone(),
```

Comment:
```suggestion
                        initdata: input,
```

It seems unnecessary to clone it